src/lib/components/pages/judge/index.tsx

```diff
@@ -0,0 +1,128 @@
+"use client";
+
+import { useState } from "react";
+
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "../../ui/dialog";
+
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "../../ui/form";
+import { Input } from "../../ui/input";
+import { Label } from "../../ui/label";
+import { Button } from "../../ui/button";
+
+interface IListTeam {
+  id: number;
+  name: string;
+  isValidate: boolean;
+}
+
+const CardTeam = (props: { name: string; isValidate: boolean }) => {
+  const { name, isValidate } = props;
+  return (
+    <div className="py-5 px-6 rounded-lg shadow-md border flex justify-between items-center">
+      <span>{name}</span>
+      <span
+        className={`p-2 ${
+          isValidate ? "bg-green-600" : "bg-red-600"
+        } text-white rounded-xl text-xs`}
+      >
+        {isValidate ? "Validated" : "Not Validated"}
+      </span>
+    </div>
+  );
+};
+
+const data: { teams: IListTeam[] } = {
+  teams: [
+    {
+      id: 1,
+      name: "Team 1",
+      isValidate: true,
+    },
+    {
+      id: 2,
+      name: "Team 2",
+      isValidate: false,
+    },
+  ],
+};
+
+const ListTeamBooth = () => {
+  const [dialogTeam, setDialogTeam] = useState<IListTeam | null>(null);
+  const [openDialog, setOpenDialog] = useState<boolean>(false);
+
+  const handleSubmitValidate = (e: any) => {
```

Comment:
you can use React.FormEvent<HTMLFormElement> instead of use any
ref: https://dev.to/fatehmohamed14/typescript-event-types-for-react-3epa